pixi.toml

```
@@ -1,7 +1,7 @@
 [project]
 name = "metalearners"
 channels = ["conda-forge"]
-platforms = ["linux-64", "osx-arm64", "osx-64", "win-64"]
+platforms = ["linux-64", "osx-arm64", "osx-64"]
```

@pavelzw For some reason `causalml` cannot be installed for `"win-64"`. Do you know of a way to disable this very platform only for the `benchmark` environment?

---

```suggestion
platforms = ["linux-64", "osx-arm64", "osx-64", "win-64"]
```

see my comment below

---

Thanks a bunch :)